OracleIdentityGovernance/README.md

```diff
@@ -0,0 +1,128 @@
+
+OIG requires a database to store the configuration information and RCU schema information. If you do not have a database available and require one for testing, then you can use an Oracle Database Docker image. The instructions below show how to install the database image and start the container.
+ 
+### Pulling the Oracle Database Image
+
+1. Launch a browser and access the [Oracle Container Registry](https://container-registry.oracle.com/).
+1. Click **Sign In** and login with your username and password.
+1. In the **Search** field enter **enterprise** and press **Enter**.
+1. Click **enterprise** Oracle Database Enterprise Edition.
+1. In the **Terms and Conditions** box, select Language as **English**. Click **Continue** and ACCEPT "**Terms and Restrictions**".
+1. On your Docker environment login to the Oracle Container Registry and enter your Oracle SSO username and password when prompted:
+
+   ```
+   $ docker login container-registry.oracle.com
+   Username: <username>
+   Password: <password>
+   ```
+   
+   For example:
+   
+   ```
+   $ docker login container-registry.oracle.com
+   Username: joe.bloggs@example.com
+   Password:
+   Login Succeeded 
+   ```
+   
+1. Pull the Oracle Database image:
+  
+   ```
+   $ docker pull container-registry.oracle.com/database/enterprise:12.2.0.1
+   ```
+   
+   The output will look similar to the following:	
+   
+   ```
+   Trying to pull repository container-registry.oracle.com/database/enterprise ...
+   12.2.0.1: Pulling from container-registry.oracle.com/database/enterprise
+   f07cd347d7cc: Pull complete
+   e6d45c5d2f56: Pull complete
+   0c3e3e3a81c6: Pull complete
+   522e6a16038b: Pull complete
+   b49278619f9a: Pull complete
+   Digest: sha256:1f700299f7a96c5ffcdb14e251745f1cf3832fc32fff59ee7fdce956bd5b5bf8
+   Status: Downloaded newer image for container-registry.oracle.com/database/enterprise:12.2.0.1
+   ```
+	
+1. Run the `docker tag` to tag the image as follows:
+
+   ```
+   [dockeruser@mydockerhost]$ docker tag container-registry.oracle.com/database/enterprise:12.2.0.1 localhost/oracle/database:12.2.0.1-ee
+   ```
+	
+1. Run the `docker images` command to show the image is installed into the repository. The output should look similar to this:	
+	
+   ```
+   $ docker images
+   REPOSITORY                                        TAG           IMAGE ID      CREATED       SIZE
+   container-registry.oracle.com/database/enterprise 12.2.0.1      12a359cd0528  3 years ago   3.44GB
+   localhost/oracle/database                         12.2.0.1-ee   12a359cd0528  3 years ago   3.44GB
+   ```
+
+1. Pull the Oracle Instant Client from the [Oracle Container Registry](https://container-registry.oracle.com/) (search for **instantclient**, click, and then accept the T&C before running the command).  This is required so you can change the default database password when you create the database later.
```

Comment:
The Instant Client image on OCR is too old to recommend. Please remove this and recommend building the Instant Client using the Dockerfile in this repo to create an up-to-date and secure container image.